adhocracy4/comments/static/comments/CommentList.jsx

```diff
@@ -22,6 +22,7 @@ const CommentList = (props) => {
               is_deleted={comment.is_deleted}
               is_removed={comment.is_removed}
               is_censored={comment.is_censored}
+              is_blocked={comment.is_blocked}
```

Comment:
This is also for the old comments, so we don't add it, right?

Reply:
If the old comments will be entirely replaced anyway, then I will take it all out.